modules/local/fodf/bundleparc/main.nf

```diff
@@ -0,0 +1,43 @@
+process BUNDLEPARC {
+    tag "$meta.id"
+    label 'process_single'
```

Comment:
Can bundleparc parallelize ? If so, change this label for `process_high` and use `task.cpus` in the script to control allocation.

Also, what is the RAM consumption expected ? Or awaited ? We can have another label to control for that.

Reply:
It can't unfortunately